index.html

```diff
@@ -13,15 +13,26 @@
 
 <body>
     <div class="container">
```

Comment:
Great job with marking up and styling your header. Just to keep your page more semantic I'd use a `<header>` tag in place of the `<div>`